docs/dev/libvirt-howto.md

```diff
@@ -11,7 +11,7 @@ Before you begin, install the [build dependencies](dependencies.md).
 On Fedora, CentOS/RHEL:
 
 ```sh
-sudo yum install libvirt-daemon
+sudo yum install libvirt-daemon libvirt-daemon-config-network
```

Comment:
Why not change this to read instead:

```
sudo yum install libvirt
```

?  That should pull in all of the configs, storage etc.

Reply:
@cgwalters : happy to do that 
@wking @rphillips : what do you think?

Reply:
>     sudo yum install libvirt
>
> ? That should pull in all of the configs, storage etc.

It's surprising to me that `libvirt-daemon` doesn't pull in this configuration :p.  While we're updating this, we probably also need `libvirt-devel`, because the destroy logic needs to link the C library.

Reply:
@wking so :

`sudo yum install libvirt libvirt-devel`